Sources/Tokenizer/Layout/Constraints/Constraint.swift

```diff
@@ -38,7 +38,16 @@ public struct Constraint {
     public static func constraints(name: String, attribute: XMLAttribute) throws -> [Constraint] {
         let layoutAttributes = try LayoutAttribute.deserialize(name)
         let tokens = Lexer.tokenize(input: attribute.text)
-        return try layoutAttributes.flatMap { try ConstraintParser(tokens: tokens, layoutAttribute: $0).parse() }
+        var constraints = try layoutAttributes.flatMap { try ConstraintParser(tokens: tokens, layoutAttribute: $0).parse() }
```

Comment:
You mean like this: 
```swift 
indexedConstraint.field = field + ".\(constraint.attribute)"
```
(outputs eg: "progressFill.right")
Or I was thinking I could add a property `multipleConstraint: Bool` and add the suffix in `LiveUIApplier.swift`.
Using tuple would maybe unnecessarily duplicate the `field` and `attribute` ... ?

Reply:
I'm not sure about the inner workings of ReactantUI in this area, but I'm thinking about creating a tuple so that when you want to reference a constraint in Swift code, you don't have to check whether you're working with right or left variant by using indices and using `progressFill.left` instead.

Again, not sure if it's even possible if RUI is not ready for this kind of approach.

Reply:
Hmm, tuple solution would be great, but AFAICT it would not be very easy to do. Maybe first fix this bug and then create another issue marked as enhancement ... ?